KIPs/kip-29.md

```diff
@@ -0,0 +1,63 @@
+---
+kip: 29
+title: Add DAO multisig to the smart contract strategy for converting & receiving LP fees
+status: Active
+type: Updating smart contract strategy
+authors: Kyber Network team
+created: 2023-12-11
+---
+
+```
+campaign_type: general
+executor: long timelock executor
+voting_power: current staked voting power
+execution: Add DAO multisig to the smart contract strategy for converting & receiving LP fees
+start_timestamp: 1702368000 (Tuesday, December 12, 2023 3:00:00 PM GMT+7)
+end_timestamp: 1702972800 (Tuesday, December 19, 2023 3:00:00 PM GMT+7)

```

Comment:
Please change this, it won't work as 3pm December 19 is the next epoch, should be:
2pm GMT+7 December 12 -> 2pm GMT+7 December 19